src/libsyntax/ext/tt/macro_parser.rs

```diff
@@ -249,22 +249,6 @@ pub enum ParseResult<T> {
 pub type NamedParseResult = ParseResult<HashMap<Ident, Rc<NamedMatch>>>;
 pub type PositionalParseResult = ParseResult<Vec<Rc<NamedMatch>>>;
 
-pub fn parse_or_else(sess: &ParseSess,
```

Comment:
Sure, I choose to remove it because 

1. It was only called in one place
2. It only forwarded it's arguments to `parse` and panicked on failure

Taking this into account I don't think the existence of this function is really justified, especially since I would have had to add another parameter to it.